test/prototype/test_low_bit_optim.py

```diff
@@ -85,7 +89,9 @@ def test_bf16_stochastic_round(self, device, compile):
         x_rep = x.view(-1, 1).repeat(1, 100_000)
 
         if compile:
-            x_rep_bf16 = torch.compile(_fp32_to_bf16_sr, fullgraph=True, dynamic=False)(x_rep)
+            x_rep_bf16 = torch.compile(_fp32_to_bf16_sr, fullgraph=True, dynamic=False)(
+                x_rep
+            )
```

Comment:
When formatting goes wrong 😢. I will fix it in #1217 (we can merge this one first)

Reply:
![image](https://github.com/user-attachments/assets/b6f90b9e-22e1-410f-b473-5591c0407332)